yoyodyne/models/base.py

```diff
@@ -305,11 +305,17 @@ def validation_epoch_end(self, validation_step_outputs: Dict) -> Dict:
         Returns:
             Dict: averaged metrics over all validation steps.
         """
-        keys = validation_step_outputs[0].keys()
-        # Shortens name to do comprehension below.
-        # TODO: there has to be a more elegant way to do this.
-        V = validation_step_outputs
-        metrics = {k: sum([v[k] for v in V]) / len(V) for k in keys}
+        num_steps = len(validation_step_outputs)
+        avg_val_loss = (
+            sum([v["val_loss"] for v in validation_step_outputs]) / num_steps
+        )
+        num_correct = sum([v["val_correct"] for v in validation_step_outputs])
```

Comment:
You don't need the `[]` inside the `sum` expression:

```python
num_correct = sum(v["val_correct"] for v in validation_step_outputs)
```

And same thing on the follwing lnie.